test/lru_hash_test.rb

```diff
@@ -58,13 +58,14 @@ def test_get_moves_the_item_at_the_top
   end
 
   def test_set_cleans_resources_if_last_error_has_expired
-    @lru_hash.set("b", create_circuit_breaker("b", true, false, 1000))
+    lru_hash = LRUHash.new(max_size: 0, min_time: 1)
+    lru_hash.set("b", create_circuit_breaker("b", true, false, 2))
 
-    Timecop.travel(2000) do
-      @lru_hash.set("d", create_circuit_breaker("d"))
+    sleep(3)
```

Comment:
I wrote small mock solution for travel. it would require either both Time.now and Monotonic time mocking or merge changes from https://github.com/Shopify/semian/pull/441 to single PR -> remove any Time.now usage from semian.

Reply:
Introduced Time helper to freeze time with +/-duration. 
Because circuit breaker requires Time.now for now and LRU hash uses Monotnic did universal stub mechanism for both clocks:
https://github.com/Shopify/semian/pull/443/files#diff-4063630c1592331c02bf7c3561f5eb835eaff847cc9d3bd993ea2488e3492cb3R3